### ISSUE TITLE
System.ValueTuple is not defined for .NET 4.0

### ISSUE THREAD

**Issue by NN---** [theraot/Theraot#39]

**Comment by theraot** [comment]
I wrote AvailabilityTests intending to prevent this kind of thing :S

I had a look at what reference got added in TestRunner, it says it is a ValueTuple for Profile36 (this is a legacy portable library profile), it seems to be a build for Silverlight (as I said, legacy), which is not listed from the NuGet.

I tried in a different project, and I had to add a reference to System.Runtime to make it work. Oddly, not from a NuGet. I tried to look from where it comes, and I think it is from one for the SDKs installed on my machine, Windows Phone, I think, will double check.

**Comment by theraot** [comment]
@NN--- I have double checked, I have tried in a new project, I have removed and added the references back into TestRunner...

At the end I opened up system.valuetuple.4.5.0.nupkg and compared.

I have to say, system.valuetuple.4.5.0.nupkg (unzip) support .NET 4.0, it has Profile36 (portable-net40+sl4+win8+wp8) (see [Target frameworks](https://docs.microsoft.com/en-us/nuget/reference/target-frameworks) for reference).

I also checked that it is not just me, see [ValueTuple availability on various platforms](http://blog.monstuff.com/archives/2017/03/valuetuple-availability.html).

---

So... are you having trouble adding System.ValueTuple on .NET 4.0?
